app/controllers/application_controller.rb

```diff
@@ -6,6 +6,12 @@ class ApplicationController < ActionController::Base
 
   helper :all
 
+  rescue_from 'ActiveRecord::SubclassNotFound' do
+    @undfined_agent_types = current_user.undfined_agent_types
```

Comment:
Thanks, the new GitHub interface is strange it does not hide comments on old diffs anymore.